backend/app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,14 @@
+class UserMailer < Devise::Mailer
+  include Devise::Controllers::UrlHelpers
+
+  default template_path: "user_mailer"
+  default from: "StudyCircle <studycircle.project@gmail.com>"
+
+  def reset_password_instructions(record, token, opts = {})
+    @token = token
+    # Agregar ENV["FRONTEND_RESET_PASSWORD_URL"] y cambiarlo
+    @frontend_url = "http://localhost:5173/restablecer_contrasena?reset_password_token=#{token}"
```

Comment:
```suggestion
    # Agregar ENV["FRONTEND_RESET_PASSWORD_URL"] y cambiarlo
    @frontend_url = "#{ENV.fetch("FRONTEND_ORIGIN", "http://localhost:5173")}/restablecer_contrasena?reset_password_token=#{token}"
```

jajaja  creo que te fallto  cambiarlo... 
Ya tenemos una env de frontend origin la podes reutilizar :)